app/views/documents/edit.html.erb

```diff
@@ -48,12 +45,10 @@
           Upload documents
         </button>
 
-        <%= link_to next_path, class: "button" do %>
-          I'll do this later
-        <% end %>
+        <button class="button button--next button--cta button--full-width" type="submit">
+          Done uploading documents
```

Comment:
If we disable it there's no way for them to continue without uploading docs (eg: "Skip")